include/services/daal_defines.h

```diff
@@ -104,6 +104,10 @@
 #endif
 #if (defined(__SYCL_COMPILER_VERSION) && (__SYCL_COMPILER_VERSION >= 20191024))
   #define DAAL_SYCL_INTERFACE_REVERSED_RANGE
+#elif (defined(COMPUTECPP_VERSION_MAJOR) && (COMPUTECPP_VERSION_MAJOR >= 1) && \
+                                            (COMPUTECPP_VERSION_MINOR >= 1) && \
+                                            (COMPUTECPP_VERSION_PATCH >= 6))
+    #define DAAL_SYCL_INTERFACE_REVERSED_RANGE
```

Comment:
I see different indentations at lines 110 and 106, which one is right?

Reply:
The last one. All the other indentations in the file contain 4 spaces. I fixed indentation on lines 106 and 103.